app/app.go

```diff
@@ -98,13 +99,24 @@ import (
 	evmkeeper "github.com/tharsis/ethermint/x/evm/keeper"
 	evmtypes "github.com/tharsis/ethermint/x/evm/types"
 
+	"github.com/peggyjv/gravity-bridge/module/x/gravity"
+	gravitykeeper "github.com/peggyjv/gravity-bridge/module/x/gravity/keeper"
+	gravitytypes "github.com/peggyjv/gravity-bridge/module/x/gravity/types"
+
 	// this line is used by starport scaffolding # stargate/app/moduleImport
 	cronosmodule "github.com/crypto-org-chain/cronos/x/cronos"
 	cronosmodulekeeper "github.com/crypto-org-chain/cronos/x/cronos/keeper"
 	cronosmoduletypes "github.com/crypto-org-chain/cronos/x/cronos/types"
 )
 
-const Name = "cronos"
+const (
+	Name = "cronos"
+
+	// AddrLen is the allowed length (in bytes) for an address.
+	//
+	// NOTE: In the SDK, the default value is 255.
+	AddrLen = 20
```

Comment:
why changing this? in 0.43, there are 32-byte addresses (e.g. if one uses a different signing algorithm)?

Reply:
https://github.com/PeggyJV/gravity-bridge/blob/main/module/app/app.go#L104

gravity seems to require this, not sure if it works without it. I guess it need to map any potential cosmos address to ethereum.

Reply:
https://github.com/PeggyJV/gravity-bridge/issues/143

Opened issue to gravity, I guess it's possible to only do the verification at module level.